packages/@dataform/core/BUILD

```diff
@@ -31,7 +31,8 @@ npm_package_bin(
     ],
     args = [
         "--config=$(location webpack.config.js)",
-        "--output=$(location bundle_no_license.js)",
+        "--output-path=$(@D)",
```

Comment:
Could you please comment here what `@D` means. Maybe add link to the docs - https://bazel.build/reference/be/make-variables

BTW, according to this link it is recommended to prefer `RULEDIR` over `@D` but maybe in Bazel 5.4.0 it wasn't introduced yet, need to check